spec/acceptance/nodesets/archlinux-2-x64.yml

```diff
@@ -0,0 +1,13 @@
+---
```

Comment:
These nodesets are probably not required.  See https://github.com/voxpupuli/puppet-redis/pull/299#discussion_r264323948

Reply:
So I should remove that file?

Reply:
You should also mark it as delete in modulesync. From the same linked repo: https://github.com/voxpupuli/puppet-redis/blob/d0ee587d0d0327ef85d5701fbd6c9d9f33569fbe/.sync.yml#L17-L30

Reply:
Read the comment and followed the example therein.

Reply:
Okay; I'll add the other lines, too.